updatecli/updatecli.d/golang.yaml

```
@@ -63,3 +63,12 @@ targets:
         'golang:{{ source `latestGoVersion` }}'
     scmID: default
 
+pullrequests:
+  default:
+    kind: github
+    scmID: default
+    targets:
+      - updateGomod
```

```suggestion
      - updateGomod
      - updateMakefile
      - updateJenkinsfile
```

Alas, until https://github.com/updatecli/updatecli/issues/445 is accepted + contributed, we have to manually manage the exhaustive list of targets.